framework/src/bundle/BundleManifest.cpp

```diff
@@ -74,14 +74,14 @@ Any ParseJsonValue(const Json::Value& jsonValue, bool ci)
   {
     return Any(jsonValue.asBool());
   }
-  else if (jsonValue.isDouble())
-  {
-    return Any(jsonValue.asDouble());
-  }
   else if (jsonValue.isIntegral())
   {
     return Any(jsonValue.asInt());
   }
+  else if (jsonValue.isDouble())
+  {
+    return Any(jsonValue.asDouble());
+  }
 
   return Any();
```

Comment:
Unrelated to this change but the number of return statements in this function makes me uncomfortable.